### ISSUE TITLE
ML/DL/RL - Summary

### ISSUE THREAD

**Issue by junxnone** [junxnone/AI#107]
- ~~**基础知识**~~
  - ~~矩阵~~
  - ~~[Convolution - 卷积](https://github.com/junxnone/tech-io/issues/153)~~
  - ~~[Gradient Descent](https://github.com/junxnone/ml/issues/89)~~
  - ~~CNN~~
  - ~~RNN~~
  - ~~[Metric Learning](https://github.com/junxnone/tech-io/issues/610)~~
- ~~**Tasks**~~
  - ~~**Classification** [[Summary](https://github.com/junxnone/ml/issues/111)] [[History](https://github.com/junxnone/tech-io/issues/996)]~~
  - ~~**Detection** [[Summary](https://github.com/junxnone/tech-io/issues/213)] [[History]()]~~
  - ~~**Segmentation** [[Summary](https://github.com/junxnone/tech-io/issues/465)] [[History]()]~~
    - ~~[Instance Segmentation](https://github.com/junxnone/tech-io/issues/305)~~
    - ~~[Semantic Segmentation]~~
  - ~~[OCR/TextDetection/TextRecognition](https://github.com/junxnone/tech-io/issues/749)~~
  - ~~[CBIR](https://github.com/junxnone/tech-io/issues/369)~~
  - ~~[3D - Point Cloud](https://github.com/junxnone/tech-io/issues/976)~~
- ~~**Tools**~~
  - ~~[Pytorch](https://github.com/junxnone/tech-io/issues/311)~~
  - ~~[Tensorflow]~~
  - ~~[OpenVINO](https://github.com/junxnone/tech-io/issues/17)~~
  - ~~[Active Learning Tools](https://github.com/junxnone/tech-io/issues/325)~~
  - ~~[Label Tools](https://github.com/junxnone/ml/issues/12)~~
- **Research Tech**
  - ~~[Transformers](https://github.com/junxnone/tech-io/issues/937)~~
  - ~~[Active Learning](https://github.com/junxnone/tech-io/issues/794)~~
  - ~~[AutoML](https://github.com/junxnone/tech-io/issues/107)~~
    - ~~[Bayesian Optimization](https://github.com/junxnone/tech-io/issues/684)~~
  - ~~[Attention](https://github.com/junxnone/tech-io/issues/757)~~
  - ~~[Knowledge Distillation](https://github.com/junxnone/tech-io/issues/960)~~
  - ~~[AutoEncoder](https://github.com/junxnone/tech-io/issues/632)~~
- ~~[**Datasets**](https://github.com/junxnone/tech-io/issues/30)~~
- ~~[**Metrics**](https://github.com/junxnone/tech-io/issues/279)~~
- [**Application**](https://github.com/junxnone/ml/issues/109)
- ~~[**Resource**](https://github.com/junxnone/ml/issues/108)~~
